File: simple-calculator/java/SimpleCalculator.java
import java.util.Scanner;

public class SimpleCalculator {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Enter the first number: ");
        var firstNumber = scanner.nextInt();
        
        System.out.print("Enter the second number: ");
        var secondNumber = scanner.nextInt();
        
        System.out.print("Choose operation from + - * / % : ");
        var operation = scanner.next();

        switch (operation) {
            case "+":
                System.out.println(firstNumber + secondNumber);
                break;
            case "-":
                System.out.println(firstNumber - secondNumber);
                break;
            case "*":
                System.out.println(firstNumber * secondNumber);
                break;
            case "/":
                if (secondNumber == 0) {
                    System.out.println("cannot divide by zero");
                    break;
                }
                System.out.println(firstNumber / secondNumber);
                break;
            case "%":
                if (secondNumber == 0) {
                    System.out.println("cannot divide by zero");
                    break;
                }
                System.out.println(firstNumber % secondNumber);
                break;
        }

        scanner.close();
    }    
}

File: simple-calculator/python/SimpleCalculator.py
print('Enter first number: ')
first_number = int(input());

print('Enter second number: ')
second_number = int(input())

print("Choose operation from + - * / % : ")
operation = input()

# --- Note --- 
# match case keywords are available only in Python 3.10 and above. Use nested if-else statements in earlier versions.
match operation:
    case '+':
        print(first_number + second_number)
    case '-':
        print(first_number - second_number)
    case '*':
        print(first_number * second_number)
    case '/':
        if (second_number == 0):
            print('cannot divide by zero')
        else:    
            print(first_number / second_number)
    case '%':
        if (second_number == 0):
            print('cannot divide by zero')
        else:
            print(first_number % second_number)
    case _:
        print('Invalid operation')